docs/pages/web/datepicker.js

```diff
@@ -128,10 +128,8 @@ function DatePickerExample() {
 
       <MainSection name="Variants">
         <Example
-          description="
-    Use DatePicker to select date inputs.
-  "
-          name="Example: Basic Date Picker"
+          description="Use DatePicker to select date inputs."
+          name="Basic Date Picker"
```

Comment:
As I was checking dark mode error text, I noticed these example names and removed the repetitive Example: in front of all of them for cleaner docs